symplyphysics/laws/chemistry/potential_energy_models/lennard_jones_potential.py

```diff
@@ -0,0 +1,53 @@
+from sympy import Eq
+from symplyphysics import (
+    units,
+    Quantity,
+    Symbol,
+    print_expression,
+    validate_input,
+    validate_output,
+)
+
+# Description
+## The Lennard-Jones (LJ) potential is an intermolecular pair potential. It models soft repulsive and
+## attractive (van der Waals) interactions, and it describes electronically neutral atoms or molecules.
+## It is a simple yet realistic model to describe intermolecular interactions: two particles repel each
+## other at a very close distance, attract each other at moderate distance, and do not interact at infinite
+## distance.
+
+# Law: U = 4*e*((sigma/r)**12 - (sigma/r)**6)
+## U - Lennard-Jones potential
+## e - depth of potential well ("dispersion energy")
+## sigma - distance at which the potential energy is zero ("particle size")
+## r - distance between interacting particles
+
+potential = Symbol("potential", units.energy)
+dispersion_energy = Symbol("dispersion_energy", units.energy)
+particle_size = Symbol("particle_size", units.length)
+distance = Symbol("distance", units.length)
+reduced_distance = distance / particle_size
+
+law = Eq(potential, 4 * dispersion_energy * (reduced_distance**(-12) - reduced_distance**(-6)))
```

Comment:
Why did you invert the formula? Now it does not look like the law in description.

Reply:
Simply remove reduced distance and write law in the same form as in description.